pkg/sql/catalog/descs/descriptor.go

```diff
@@ -33,7 +33,6 @@ func (tc *Collection) GetMutableDescriptorsByID(
 	ctx context.Context, txn *kv.Txn, ids ...descpb.ID,
 ) ([]catalog.MutableDescriptor, error) {
 	flags := tree.CommonLookupFlags{
-		Required:       true,
```

Comment:
I did a systematic cleanup of the flags which are ignored by internal method callees, so you'll see a bunch of other no-op changes like this.